### ISSUE TITLE
Empty URLs are dumped to browser console

### ISSUE THREAD

**Issue by kirill-konshin** [compedit/react-soundcloud-widget#14]
This line https://github.com/troybetz/react-soundcloud-widget/blob/master/src/SoundCloud.js#L119 is causing lots of 404 errors to be dumped to console:

```
src="https://w.soundcloud.com/player/?url="
```

You need to open `IFRAME` with some other URL that is more safe.

**Comment by strachsel** [comment]
Hey Kirill, were you able to find a workaround here? It works for one track, but when I add a couple of tracks, the rest of them don't render and throw empty urls and 404s.

**Comment by kirill-konshin** [comment]
Nope. I ended up dropping the widget, I now just add an IFRAME because I don't need to track play/pause statuses or integrate in any other way.
